src/ripple/rpc/handlers/AccountTxOld.cpp

```diff
@@ -155,8 +155,8 @@ Json::Value doAccountTxOld (RPC::JsonContext& context)
                 Json::Value& jvObj = jvTxns.append (Json::objectValue);
 
                 std::uint32_t  uLedgerIndex = std::get<2> (*it);
-                jvObj[jss::tx_blob]            = std::get<0> (*it);
-                jvObj[jss::meta]               = std::get<1> (*it);
+                jvObj[jss::tx_blob]            = strHex(std::get<0> (*it));
+                jvObj[jss::meta]               = strHex(std::get<1> (*it));
```

Comment:
I changed the return type of `getAccountTxsB()` to return a Blobs instead of strings for the binary data. This is because the gRPC handler does not stringify anything. So, converting to a hex string for the JSON API needs to happen here.